src/data/monsters.txt

```diff
@@ -154,7 +154,7 @@ conservationist hippy	597	conhippy.gif	BOSS NOCOPY Atk: 64 Def: 68 HP: 70 Init:
 cooler wino	1751	coolerwino.gif	Atk: 2 Def: 1 HP: 2 Init: -10000 P: hobo Article: a	unflavored wine cooler (100)	unflavored wine cooler (30)	mostly-broken sunglasses (30)	handful of stubble (c0)
 Copperhead Club bartender	1510	coppertender.gif	NOWISH Atk: 140 Def: 140 HP: 150 Init: 25 Meat: 100 P: dude EA: cold EA: hot EA: none	handful of tips (20)	unnamed cocktail (15)	unrequired jacket (0)	electric copperhead potion (c0)
 corpulent zobmie	196	fatzombie.gif	NOWISH Atk: 54 Def: 49 HP: 50 Init: 50 Meat: 75 P: undead E: spooky Article: a	cranberries (25)	cranberries (15)	loose teeth (10)	Grateful Undead T-shirt (c1)
-cosmetics wraith	1545	makeupwraith.gif	NOCOPY Atk: 65 Def: 55 HP: 65 Init: 50 P: undead ED: spooky EA: sleaze EA: none Article: a	Lady Spookyraven's powder puff (c100)	old eyebrow pencil (n100)	old rosewater cream (n100)	old bronzer (n100)
+cosmetics wraith	1545	makeupwraith.gif	BOSS Atk: 65 Def: 55 HP: 65 Init: 50 P: undead ED: spooky EA: sleaze EA: none Article: a	Lady Spookyraven's powder puff (c100)	old eyebrow pencil (n100)	old rosewater cream (n100)	old bronzer (n100)
```

Comment:
for ones like this, where you've removed the NOCOPY tag: can you edit MonsterData's
```
public boolean isNoCopy() {
  return this.noCopy;
}
```

to something like
```
public boolean isNoCopy() {
  // NOCOPY mimics the ingame `noputty` flag which blocks McTwist
  // BOSS blocks instakill and copying separately
  return this.noCopy || this.boss;
}
```


because bosses are still uncopyable and e.g. don't go into the backup camera queue (right?)

Reply:
yes boss tag blocks most copy sources on its own. (it should be all currently but there may have been times where a new copy source briefly forgets to test boss, or we may even get a boss-copier one day). There is one bit of weirdness that just came up in spading that I need to investigate. The crimbo 2024 monsters we're believed to have been made noputty via zone script, as they can be wished or otherwise copied normally, but today backup camera did not work on them, its possible this is a known quirk of camera i'm missing but there may be something else happening.